tools/test.py

```diff
@@ -1672,6 +1656,12 @@ def Main():
         all_cases += cases
         all_unused.append(unused_rules)
 
+  # We want to skip the inspector tests if node was built without the inspector.
+  has_inspector = Execute([vm,
+      "-p", "process.config.variables.v8_enable_inspector"], context)
+  if has_inspector.stdout.rstrip() == "0":
+      context.v8_enable_inspector = False
```

Comment:
Does this set `context.v8_enable_inspector` if `process.config.variables.v8_enable_inspector` is falsy?

Reply:
Only if it's specifically `0`, not just generally falsy.

Reply:
I guess this is okay as long as the only valid values for `process.config.variables.v8_enable_inspector` are `0` and non-zero.

Reply:
AFAICT possible values are `0`, `1` with `undefined` in v6 and v4.

Reply:
For v6 it's `process.config.variables.v8_inspector`

Reply:
Okay, I'd be inclined to leave this as is here, and just not backport to 4.x and 6.x. Seem reasonable?

Reply:
The V8 inspector support isn't in v4.x so no need to backport there.